documentation/versioned_docs/version-1.6.0/License and Bill of Materials

```diff
@@ -0,0 +1,17 @@
+---
+id: License and Bill of materials
+sidebar_position: 6
+title: License and Bill of material?
+slug: /
+---
+
+Monaco is an open-source project released under the [Apache 2.0 license](https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/blob/main/LICENSE).
+
+## Where can I find the Bill of Material for Monaco?
+
+To download a copy of the bill of material for Monaco:
+1. click [here](https://github.com/keptn/keptn/actions/runs/1416203195)
+2. click on "generate-dependecies"
```

Comment:
I don't know if a regular user in GitHub can trigger a workflow - but I doubt it. How about the following: We trigger the workflow once before any monaco release and then put the link to the workflow run into the respective release documentation? Then a user of e.g. monaco version 1.6 just needs to download the files from the documentation. What do you think about that workflow @DavidPHirsch?

Reply:
@didiladi yes, that could also work 👍🏻

Reply:
@didiladi I just checked and workflows can be triggered by anyone even if they do not have a Github account. I would keep the changes I made for now, but we can definitely improve it later.

Reply:
@DavidPHirsch cool, I wasn't aware of that. Thanks for checking! Ok, then you can ignore my comment.